src/app/globals.css

```
@@ -250,51 +251,84 @@ html {
   font-family: var(--font-deck-pixel), monospace;
 }
 
-/* Retro CTA — “pressed” on hover (shadows must live in CSS so :hover can override inline styles) */
+/* Pixel-art retro game buttons — rounded bevel style */
 .home-page-root .home-cta-link {
   --cta: 280 80% 52%;
+  --cta-light: hsl(var(--cta) / 0.55);
+  --cta-dark: hsl(var(--cta));
+  position: relative;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)